dask_sphinx_theme/layout.html

```diff
@@ -1,5 +1,33 @@
 {%- set favicon_url = favicon_url or ('_static/' + ('images/favicon.svg')) %} {%
-extends "sphinx_book_theme/layout.html" %} {%- block content %}
+extends "sphinx_book_theme/layout.html" %} {% block extrahead %}
+<!-- Google Tag Manager -->
+<script>
+  (function (w, d, s, l, i) {
+    w[l] = w[l] || [];
+    w[l].push({
+      "gtm.start": new Date().getTime(),
+      event: "gtm.js",
+    });
+    var f = d.getElementsByTagName(s)[0],
+      j = d.createElement(s),
+      dl = l != "dataLayer" ? "&l=" + l : "";
+    j.async = true;
+    j.src = "https://www.googletagmanager.com/gtm.js?id=" + i + dl;
+    f.parentNode.insertBefore(j, f);
+  })(window, document, "script", "dataLayer", "GTM-P4GQM59");
```

Comment:
I guess `GTM-P4GQM59` is our ID? Do we want to make this overridable via the Sphinx theme config? Do you see any reason why we might want to change this on a per-project basis in the future?